test/controllers/votes_controller_test.rb

```diff
@@ -0,0 +1,57 @@
+require "test_helper"
+
+describe VotesController do
+  describe "create" do
+    let(:work) {
+      works(:custom1)
+    }
+
+    let(:user) {
+      User.first
+    }
+
+    let (:login) {
+      login_data = {
+        user: {
+          username: User.first.username,
+        },
+      }
+      post login_path, params: login_data
+    }
+
+    let (:vote) {
+      votes(:custom1)
+    }
+
+    it "creates a new vote" do
+      login
+
+      expect {
+        post work_votes_path(work.id)
+      }.must_change "Vote.count", 1
+      must_redirect_to root_path
```

Comment:
This test is failing because creating a new vote redirects you to the work's show path.